stream_chat/tests/test_client.py

```diff
@@ -328,7 +328,7 @@ def wait():
 
         custom = {
             "name": name,
-            "resource": "DeleteChannel",
+            "action": "DeleteChannel",
```

Comment:
@AnatolyRugalev how about async tests?

Reply:
Yup, missed that one and I merged it too early overall. Will follow up with fixes asap